bindings-go/hack/tools.go

```diff
@@ -0,0 +1,16 @@
+// +build tools
+
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// This package imports things required by build scripts, to force `go mod` to see them as dependencies
+package tools
+
+import (
+	_ "github.com/onsi/ginkgo/ginkgo"
+	_ "golang.org/x/lint/golint"
+	_ "github.com/golang/mock/mockgen"
```

Comment:
I think we can remove the mockgen from this repo, as we currently do not generate any mocks here

Reply:
changed with latest commit